sds_data_manager/constructs/batch_compute_resources.py

```diff
@@ -168,7 +168,10 @@ def __init__(
             type="CONTAINER",
             platform_capabilities=["FARGATE"],
             container_properties={
-                "image": f"{repo.repository_uri}:{account_name}",
+                "image": f"{repo.repository_uri}:latest",
+                "environment": [
+                    {"name": "SPICE_DIRECTORY", "value": "/mnt/spice"},
```

Comment:
Set this with IMAP as a prefix which is sometimes nice to grep through environment variables during processing.
```suggestion
                    {"name": "IMAP_SPICE_DIR", "value": "/mnt/spice"},
```